### ISSUE TITLE
NullReferenceException on start

### ISSUE THREAD

**Issue by ZdenekM** [robofit/arcor2_editor#76]
```
NullReferenceException: Object reference not set to an instance of an object
TrackingManager.Start () (at Assets/TABLET/Scripts/AR/TrackingManager.cs:38)
```

**Comment by Kapim** [comment]
Temporarily fixed by #80. @xBambusekD can you tak a look on why tracking manager is being instantioned in standalone version?

**Comment by Kapim** [comment]
Moving to the next release, the temp fix is OK for now.

**Comment by Kapim** [comment]
Fixed in #83
